scripts/fish_sample_build.R

```diff
@@ -25,59 +25,59 @@ fish_coll <- fpr::fpr_import_hab_con(backup = F, row_empty_remove = T) %>%
-                surveyor_3 = NA_character_,
-                camera_id = NA_character_,
-                gps_id = NA_character_,
-                gps_waypoint_number = NA_character_,
-                photo_site_bottom = NA_character_,
-                photo_site_top = NA_character_
-  ) %>%
-  # populate utm fields with a location so we can use to burn geopackage
-  mutate(utm_easting = 687879,
-         utm_northing = 6020659) %>%
-  # make it a spatial file so we can burn it as a geopackage right into our mergin file of choice
-  sf::st_as_sf(coords = c("utm_easting", "utm_northing"),
-               crs = 32600 + utm_zone, remove = F) %>%
-  # put it into albers so we can use it anywhere
-  sf::st_transform(crs = 3005) %>%
-  relocate(date_time_start,
-           mergin_user,
-           contains('surveyor'),
-           contains('name'),
-           matches('utm')) %>%
-  relocate(matches('photo'), .after = last_col()) %>%
-  # burn to test project for now, will burn to dff when finalised
-  sf::st_write(paste0('../../gis/',
-                      dir_project,
-                      '/',
-                      file_name_site,
-                      '.gpkg'),
-               # turned this T now that we have time in name
-               delete_layer = T)
+# form_prep1 <- fish_site %>%
+#   # add some columns of our own
+#   dplyr::mutate(date_time_start = NA_POSIXct_,
+#                 utm_easting = NA_integer_,
+#                 utm_northing = NA_integer_,
+#                 mergin_user = NA_character_,
+#                 surveyor_1 = NA_character_,
+#                 surveyor_2 = NA_character_,
+#                 surveyor_3 = NA_character_,
+#                 camera_id = NA_character_,
+#                 gps_id = NA_character_,
+#                 gps_waypoint_number = NA_character_,
+#                 photo_site_bottom = NA_character_,
+#                 photo_site_top = NA_character_
+#   ) %>%
+#   # populate utm fields with a location so we can use to burn geopackage
+#   mutate(utm_easting = 687879,
+#          utm_northing = 6020659) %>%
+#   # make it a spatial file so we can burn it as a geopackage right into our mergin file of choice
+#   sf::st_as_sf(coords = c("utm_easting", "utm_northing"),
+#                crs = 32600 + utm_zone, remove = F) %>%
+#   # put it into albers so we can use it anywhere
+#   sf::st_transform(crs = 3005) %>%
+#   relocate(date_time_start,
+#            mergin_user,
+#            contains('surveyor'),
+#            contains('name'),
+#            matches('utm')) %>%
+#   relocate(matches('photo'), .after = last_col()) %>%
+#   # burn to test project for now, will burn to dff when finalised
+#   sf::st_write(paste0('../../gis/',
+#                       dir_project,
+#                       '/',
+#                       file_name_site,
+#                       '.gpkg'),
+#                # turned this T now that we have time in name
+#                delete_layer = T)
 
 form_prep2 <- fish_coll %>%
```

Comment:
let's burn directly to the repo vs to projects.  please file an issue and correct when you get a chance

Reply:
i see your comment there that we can do it later. glad you thought of it.  let's do it now though so that we can track the changes with git and comments as we go

Reply:
guessing we need to add the qlm from the form_fiss_site that links to the fish form for the fish form to work.  lets make an issue that this needs to be done so it is less likely to be forgotton

Reply:
reopening #12 with a note would do the trick

Reply:
You have to change your project properties and add a relation to get it to work. I left you a note a while back in #mergin-digital-data-collection channel on slack. But I'll move all the info to the reopened issue so we remember.